static/src/javascripts/projects/common/modules/commercial/sticky-top-banner.js

```diff
@@ -76,8 +76,11 @@ define([
         var $iframe = getAdIframe();
         var slotWidth = $iframe.attr('width');
         var slotHeight = $iframe.attr('height');
+        var slotSize = [slotWidth, slotHeight].join(',');
```

Comment:
DFP uses the sizes of the winning creative to set the `width` and `height` on the `iframe`? I didn't know that 😮